src/kamel.ts

```diff
@@ -14,15 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { exec, spawn} from "child_process";
+import { exec, spawn, ChildProcess} from "child_process";
 import * as child_process from "child_process";
 import * as config from './config';
 import * as utils from './CamelKJSONUtils';
 import * as extension from './extension';
 import * as path from 'path';
 import * as fs from 'fs';
-import * as shell from './shell';
+import { findBinary } from './kubectl';
```

Comment:
That's good to remove duplicated code.
I think the `findBinary` should be extracted to its own class. There is no reason that kamel has a dependency on kubectl.

Reply:
agreed. moved it to the shell.ts